src/main.js

```diff
@@ -82,6 +104,18 @@ const main = (inFocusMode = false) => {
                     }
                    `)
                 }
+                
+                if(logseq.settings.hide_topbar) {
+                    console.log('unfocus, top bar shown', logseq.settings.hide_topbar);
```

Comment:
remove

Reply:
I am not seeing the top bar re-appear on unfocus.  Is it working for you?

Reply:
Fixed the issue